string name
geometry_msgs/Pose reference_pose
string reference_object_mesh
KeyFrameMsg[] keyframes